### ISSUE TITLE
Conditionally render items in profile page

### ISSUE THREAD

**Issue by kylemh** [OperationCode/front-end#1269]
# Feature request
- https://github.com/OperationCode/front-end/blob/main/pages/profile/index.js#L16 all of these items should be re-marked as "optional" (re: not `.isRequired()`).
- The corresponding elements these props are used to render should be then used to conditionally render the UI. For example:
```jsx
{!!employmentStatus && (
  <p>
    <strong>Employment Status : </strong>
    {employmentStatus}
  </p>
)}
```

**Comment by fletcherrippon** [comment]
Can I be assigned this issue? :)

**Comment by kylemh** [comment]
We have a policy of not assigning issues to users not in our organization, to outwardly communicate that anybody is welcome to claim the issue. For context, we often dealt with people asking for assignment, but never returning. This accidental parking leads to extra maintenance work for maintainers.

Not to worry regarding Hacktoberfest though! Assuming you face "competition", I will not mark your fix as `spam` - GitHub only cares about _opened_ pull requests, not merged ones.

**Comment by fletcherrippon** [comment]
Should all of the profile properties be rendered conditionally? 
- firstName
- lastName
- branchOfService
- companyName
- companyRole
- createdAt,
- disciplines
- email
- employmentStatus
- isMentor
- militaryStatus
- programmingLanguages

even things like `firstName`, `lastName` and `email`?

**Comment by kylemh** [comment]
Good point. Things that are _required_ in our registration form should be expected to be returned in our response from the back-end.

**Comment by fletcherrippon** [comment]
Just sent a pull request to fix the issue.

**Comment by kylemh** [comment]
Resolved in #1271
